.env.example

```diff
@@ -183,3 +183,6 @@ CSP_POLICY=
 EDGE_CONFIG=
 
 NEXT_PUBLIC_MINUTES_TO_BOOK=5 # Minutes
+
+GOOGLE_RECAPTCHA_SITE_SECRET=
+NEXT_PUBLIC_GOOGLE_RECAPTCHA_SITE_KEY=
```

Comment:
instead of putting this here, maybe its better if people can bring their own tokens. See the "FATHOM" app how they implemented adding your own credentials in the event-types

Reply:
other Captcha Apps should take their own credentials, too